ajax/libs/vertx/package.json

```diff
@@ -18,5 +18,17 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/vert-x/vert.x"
```

Comment:
@PeterDaveHello Can I ask how to determine what cases should we add another commit? (or should not)